noolite_serial.py

```diff
@@ -81,15 +81,31 @@ def status(self, ch):
         self.send_command(ch, 128, 2, 0)
         pass
 
+    def bind(self, ch):
+        self.send_command(ch, 15, 2, 0)
+        pass
+
+    def unbind(self, ch):
+        self.send_command(ch, 9, 2, 0)
+        pass
+
     def send_command(self, ch, cmd, mode=0, ctr=0, res=0, fmt=0, d0=0, d1=0, d2=0, d3=0, id0=0, id1=0, id2=0, id3=0):
         command = NooLiteCommand(ch, cmd, mode, ctr, res, fmt, d0, d1, d2, d3, id0, id1, id2, id3)
         self.tty.write(command.to_bytes())
         pass
 
     def receive(self):
         all_responses = list()
-        while self.tty.inWaiting() >= 17:
-            bytes_response = list(self.tty.read(17))
+        received = 0
```

Comment:
Please move this as a separate wrapper method. Currently, receive is used in a loop alongside with MQTT processing (see main.py) and not expected to wait for data.